tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_volume_permissions.py

```diff
@@ -0,0 +1,80 @@
+import random
+import string
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    pre-requisites:
+    1. Create cephfs subvolumegroup
+    2. Create cephfs subvolume in subvolume group with default permission
+    3. Create cephfs subvolumes in subvolumegroup with different permission
+    Test operation:
+    1. Get path of all created subvolumes
+    2. Remove all the subvolumes
+    3. Remove the subvolumegroup
+    """
+    try:
+        tc = "CEPH-83574190"
+        log.info(f"Running CephFS tests for Polarion ID -{tc}")
+        fs_util = FsUtils(ceph_cluster)
+        clients = ceph_cluster.get_ceph_objects("client")
+        client1 = clients[0]
+        fs_details = fs_util.get_fs_info(client1)
+        if not fs_details:
+            fs_util.create_fs(client1, "cephfs")
+        fs_util.auth_list([client1])
+        subvol_group = "subvolume_groupname_" + "".join(
+            random.choice(string.ascii_lowercase + string.digits)
```

Comment:
Can we generate random string once and use it
vol_name_rand =  random.choice(string.ascii_lowercase + string.digits)
            for _ in list(range(5))
subvol_group = f"subvolume_groupname_{vol_name_rand}"
subvol_default = f"subvol_default_{vol_name_rand}"
subvol_different = f"subvol_different_{vol_name_rand}"

Reply:
@AmarnatReddy I have updated the changes.